app/models/manageiq/providers/openstack/infra_manager.rb

```diff
@@ -73,4 +74,36 @@ def supports_authentication?(authtype)
   def self.event_monitor_class
     ManageIQ::Providers::Openstack::InfraManager::EventCatcher
   end
+
+  def clouds
+    overclouds = ManageIQ::Providers::Openstack::CloudManager.find(:all, :conditions => ["id != ? and provider_id = ?", id, provider_id])
```

Comment:
```ruby
ManageIQ::Providers::Openstack::CloudManager.where(:provider_id => provider_id).where.not(:id => id)
```

Also
- the variable is not necessary.
- should this be a private method?
- Why build your own query at all?  It's brittle if the relationships change moving forward.  It's hard to tell in the diff, but don't you already have the provider object...

```ruby
provider.cloud_ems
```

cc @Ladas Can you verify this?